### PR TITLE
Prachi ppp

### PR DESCRIPTION
**Pull request by prachi2023**
#145

### CODE REVIEW

File: docs/team/prachi2023.md [hunk 1]
```diff
@@ -8,17 +8,61 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
...
-
+ 1. Maintain portions on the following commands 
+     - `pin-n`
+     - `view-n` 
 ### Contributions to Developer Guide
-
-### Community
-(https://nus-cs2113-ay2021s1.github.io/addressbook-level3/team/johndoe.html)
+ 1. StorageManager related class and sequence diagrams 
+ 1. PinCommand and ViewNoteCommand sequence diagram
```
Comment (R-Ramana) [inline docs/team/prachi2023.md:68]: I don't think this is what the community is for. Can refer to the link to see how it is written.

File: docs/team/prachi2023.md [hunk 2]
```diff
@@ -8,17 +8,61 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
...
+     - `CreateTagCommand`
+     - `DeleteEventCommand`
+     - `DeleteNoteCommand`
+     - `EditEventCommand`
+     - `EditNoteCommand`
+     - `PinCommand`
+     - `TagEventCommand`
+     - `TagNoteCommand`
+   
+   1. Saving of all the notes and events when the programme ends
```
Comment (R-Ramana) [inline docs/team/prachi2023.md:37]: Need to elaborate on your functions. Look at my PPP or Jun Xiang's to get an idea (PPP or PR depending on what time you see it).

File: docs/team/prachi2023.md [hunk 3]
```diff
@@ -8,17 +8,61 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Documentation: responsible for the parts regarding saving and loading of files 
+- Implementation of I/O
+- In charge of `pin-n`, `view-n`
```
Comment (R-Ramana) [inline docs/team/prachi2023.md:13]: Can include your storage as well! :)

File: docs/team/prachi2023.md [hunk 4]
```diff
@@ -8,17 +8,61 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 
 *Roles (short for ‘in charge of role’): Description*
 
-- 
+- Documentation: responsible for the parts regarding saving and loading of files 
+- Implementation of I/O
```
Comment (R-Ramana) [inline docs/team/prachi2023.md:12]: Not very clear about I/O. Maybe you can elaborate?

**Review by R-Ramana — APPROVED** [review]

File: src/main/java/seedu/notus/data/notebook/Note.java [hunk 5]
```diff
@@ -109,10 +109,6 @@ public void togglePinned() {
         isPinned = !isPinned;
     }
 
-    public void setPinned(Boolean pinned) {
```
[the remark this answers is not in the document]
Comment (prachi2023) [reply src/main/java/seedu/notus/data/notebook/Note.java:112]: Nope I created it while setting pin for pinCommand then changed to togglePin and made sure it isn't being used anywhere else

**Pull request merged**